core/src/test/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluatorTest.scala

```diff
@@ -418,12 +418,12 @@ class OpMultiClassificationEvaluatorTest extends FlatSpec with TestSparkContext
     outputMetrics.ConfMatrixThresholds shouldEqual testThresholds
     outputMetrics.ConfMatrices.length shouldEqual testThresholds.length
     // topK confusion matrix for p >= 0.4
-    outputMetrics.ConfMatrices(0) shouldEqual
+    outputMetrics.ConfMatrices(0).ConfusionMatrixCounts shouldEqual
```

Comment:
Can you add a line here verifying the threshold? Something like outputMetrics.ConfMatrices(0).Threshold shouldEqual....